test/models/user_test.rb

```diff
@@ -395,172 +394,171 @@
       end
     end
   end
 
-  describe "#read_notices / #unread_notices" do
+  describe '#read_notices / #unread_notices' do
     before { @user = create(:user) }
 
-    context "きづきがないとき" do
-      it "#read_noticesが0件であること" do
+    context 'きづきがないとき' do
+      it '#read_noticesが0件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが0件であること" do
+      it '#unread_noticesが0件であること' do
         @user.unread_notices.count.must_equal 0
       end
     end
 
-    context "きづきが1件のとき" do
+    context 'きづきが1件のとき' do
       before { @notice = create(:notice) }
 
-      it "#read_noticesが0件であること" do
+      it '#read_noticesが0件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 1
         @user.unread_notices.first.must_equal @notice
       end
     end
 
-    context "読まれたきづきが1件のとき" do
+    context '読まれたきづきが1件のとき' do
       before do
         @notice = create(:notice)
         @notice.read_by @user
       end
 
-      it "#read_noticesが1件であること" do
+      it '#read_noticesが1件であること' do
         @user.read_notices.count.must_equal 1
         @user.read_notices.first.must_equal @notice
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 0
       end
     end
 
-    context "ほかのひとに読まれたきづきが1件のとき" do
+    context 'ほかのひとに読まれたきづきが1件のとき' do
       before do
         @another_user = create(:user)
         @notice = create(:notice)
         @notice.read_by @another_user
       end
 
-      it "#read_noticesが1件であること" do
+      it '#read_noticesが1件であること' do
         @user.read_notices.count.must_equal 0
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 1
         @user.unread_notices.first.must_equal @notice
       end
     end
 
-    context "読んだきづきと読んでいないものが1件ずつあるとき" do
+    context '読んだきづきと読んでいないものが1件ずつあるとき' do
       before do
         @unread_notice = create(:notice)
         @read_notice = create(:notice)
         @read_notice.read_by @user
       end
 
-      it "#read_noticesが1件であること" do
+      it '#read_noticesが1件であること' do
         @user.read_notices.count.must_equal 1
         @user.read_notices.first.must_equal @read_notice
       end
 
-      it "#unread_noticesが1件であること" do
+      it '#unread_noticesが1件であること' do
         @user.unread_notices.count.must_equal 1
         @user.unread_notices.first.must_equal @unread_notice
       end
-
     end
   end
 
-  describe "#activities_for_heatmap" do
+  describe '#activities_for_heatmap' do
     before do
       @user = create(:user)
       Timecop.freeze
     end
 
     [:notice, :reply].each do |activity_type|
-      context "noticeがないとき" do
-        it "nilであること" do
+      context 'noticeがないとき' do
+        it 'nilであること' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.